splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/joins/BroadcastJoinIT.java

```diff
@@ -287,94 +289,97 @@ public static void createDataSet() throws Exception {
     @Test
     public void testNumericColumnsBroadCastJoin() throws Exception {
 
-        String expected = "1 |\n" +
+        String [] joinTypes = {"NestedLoop", "SortMerge", "Broadcast", "Cross" };
```

Comment:
It may be better to remove the wording "Broadcast" from the test case name as now we test all 4 join strategies.

Reply:
Updated the method name to testNumericJoinColumns.